libs/types-common/src/Wire/Timeout.hs

```diff
@@ -0,0 +1,32 @@
+module Wire.Timeout
+  ( Timeout (..),
+    module Data.Time.Clock,
+  )
+where
+
+import Data.Aeson
+import Data.Aeson.Types
+import Data.Scientific
+import Data.Time.Clock
+import Imports
+
+newtype Timeout = Timeout
+  { timeoutDiff :: NominalDiffTime
+  }
+  deriving newtype (Eq, Enum, Ord, Num, Real, Fractional, RealFrac, Show)
+
+instance Read Timeout where
+  readsPrec i s =
+    case readsPrec i s of
+      [(x :: Int, s')] -> [(Timeout (fromIntegral x), s')]
+      _ -> []
+
```

Comment:
See if we can remove it, if yes we can use this type to replace the other Timeout type.

Reply:
I just compiled everything without the FromJSON instance and it turns out we need it in `Brig.Options`.

Further, our other `Timeout` type contains a `Word64`, and this one contains `NominalDiffTime`, so we shouldn't touch this topic in this PR...